readme.md

```diff
@@ -296,6 +354,11 @@ unzip /tmp/data/berlin.zip -d /tmp/data
 
 # download openstreetmap data
 curl -s https://s3.amazonaws.com/metro-extracts.mapzen.com/berlin_germany.osm.pbf > /tmp/data/berlin.osm.pbf
+
+#download tiger data
+export TIGERPATH=/tmp/data/tiger #make sure that directory exists
```

Comment:
It's not really clear if this is a directive to the user reading the readme or a comment on what this line does. A veteran bash user will know it's a directive to the user, but I think we can make it more explicit.  Also perhaps we should use `mkdir -p` to create the directory on behalf of a user so that it doesn't matter either way.